init/arch_specs/eessi_arch_arm.spec

```diff
@@ -1,5 +1,6 @@
 # ARM CPU architecture specifications (see https://gpages.juszkiewicz.com.pl/arm-socs-table/arm-socs.html for guidance)
 # Software path in EESSI 	| Vendor ID 	| List of defining CPU features
+"aarch64/aarch64"	""		"asimdhp sve"		# Fujitsu A64FX
```

Comment:
```suggestion
"aarch64/a64fx"	""		"asimdhp sve"		# Fujitsu A64FX
```

Reply:
This wasn't the only issue, fixed both in cb1672b with force push